Description: fix manpage
 This patch fixes trailing whitespace and some minor grammatical errors
Author: Philipp Huebner <debalance@debian.org>

Index: rebar3/manpages/rebar3.1
===================================================================
--- rebar3.orig/manpages/rebar3.1
+++ rebar3/manpages/rebar3.1
@@ -49,25 +49,23 @@ Start the current project in a REPL\. Yo
 
 .SH "COMMANDS"
 
-. this section generated by running the contents of 'commands' in rebar3 shell
-
 .TP
-\fBas\fR 
+\fBas\fR
 Higher order provider for running multiple tasks in a sequence as a certain profiles.
 .TP
-\fBclean\fR [\fI-a\fR|\fI--all\fR] [\fI-p\fR|\fI--profile\fR] 
+\fBclean\fR [\fI-a\fR|\fI--all\fR] [\fI-p\fR|\fI--profile\fR]
 Remove compiled beam files from apps.
 .IP
 \fI--all\fR: Clean all apps include deps
 .IP
 \fI--profile\fR: Clean under profile. Equivalent to `rebar3 as <profile> clean`
 .TP
-\fBcompile\fR [\fI-d\fR|\fI--deps_only\fR] 
+\fBcompile\fR [\fI-d\fR|\fI--deps_only\fR]
 Compile apps .app.src and .erl files.
 .IP
 \fI--deps_only\fR: Only compile dependencies, no project apps will be built.
 .TP
-\fBcover\fR [\fI-r\fR|\fI--reset\fR] [\fI-v\fR|\fI--verbose\fR] [\fI-m\fR|\fI--min_coverage\fR] 
+\fBcover\fR [\fI-r\fR|\fI--reset\fR] [\fI-v\fR|\fI--verbose\fR] [\fI-m\fR|\fI--min_coverage\fR]
 Perform coverage analysis.
 .IP
 \fI--reset\fR: Reset all coverdata.
@@ -76,7 +74,7 @@ Perform coverage analysis.
 .IP
 \fI--min_coverage\fR: Mandate a coverage percentage required to succeed (0..100)
 .TP
-\fBct\fR [\fI--dir\fR] [\fI--suite\fR] [\fI--group\fR] [\fI--case\fR] [\fI--label\fR] [\fI--config\fR] [\fI--spec\fR] [\fI--join_specs\fR] [\fI--allow_user_terms\fR] [\fI--logdir\fR] [\fI--logopts\fR] [\fI--verbosity\fR] [\fI-c\fR|\fI--cover\fR] [\fI--cover_export_name\fR] [\fI--repeat\fR] [\fI--duration\fR] [\fI--until\fR] [\fI--force_stop\fR] [\fI--basic_html\fR] [\fI--stylesheet\fR] [\fI--decrypt_key\fR] [\fI--decrypt_file\fR] [\fI--abort_if_missing_suites\fR] [\fI--multiply_timetraps\fR] [\fI--scale_timetraps\fR] [\fI--create_priv_dir\fR] [\fI--include\fR] [\fI--readable\fR] [\fI-v\fR|\fI--verbose\fR] [\fI--name\fR] [\fI--sname\fR] [\fI--setcookie\fR] [\fI--sys_config\fR] [\fI--compile_only\fR] [\fI--retry\fR] 
+\fBct\fR [\fI--dir\fR] [\fI--suite\fR] [\fI--group\fR] [\fI--case\fR] [\fI--label\fR] [\fI--config\fR] [\fI--spec\fR] [\fI--join_specs\fR] [\fI--allow_user_terms\fR] [\fI--logdir\fR] [\fI--logopts\fR] [\fI--verbosity\fR] [\fI-c\fR|\fI--cover\fR] [\fI--cover_export_name\fR] [\fI--repeat\fR] [\fI--duration\fR] [\fI--until\fR] [\fI--force_stop\fR] [\fI--basic_html\fR] [\fI--stylesheet\fR] [\fI--decrypt_key\fR] [\fI--decrypt_file\fR] [\fI--abort_if_missing_suites\fR] [\fI--multiply_timetraps\fR] [\fI--scale_timetraps\fR] [\fI--create_priv_dir\fR] [\fI--include\fR] [\fI--readable\fR] [\fI-v\fR|\fI--verbose\fR] [\fI--name\fR] [\fI--sname\fR] [\fI--setcookie\fR] [\fI--sys_config\fR] [\fI--compile_only\fR] [\fI--retry\fR]
 Run Common Tests.
 .IP
 \fI--dir\fR: List of additional directories containing test suites
@@ -125,7 +123,7 @@ Run Common Tests.
 .IP
 \fI--abort_if_missing_suites\fR: Abort if suites are missing
 .IP
-\fI--multiply_timetraps\fR: 
+\fI--multiply_timetraps\fR:
 .IP
 \fI--scale_timetraps\fR: Scale timetraps
 .IP
@@ -149,26 +147,26 @@ Run Common Tests.
 .IP
 \fI--retry\fR: Experimental feature. If any specification for previously failing test is found, runs them.
 .TP
-\fBdeps\fR 
+\fBdeps\fR
 List dependencies
 .TP
-\fBdialyzer\fR [\fI-u\fR|\fI--update-plt\fR] [\fI-s\fR|\fI--succ-typings\fR] 
+\fBdialyzer\fR [\fI-u\fR|\fI--update-plt\fR] [\fI-s\fR|\fI--succ-typings\fR]
 Run the Dialyzer analyzer on the project.
 .IP
 \fI--update-plt\fR: Enable updating the PLT. Default: true
 .IP
 \fI--succ-typings\fR: Enable success typing analysis. Default: true
 .TP
-\fBdo\fR 
+\fBdo\fR
 Higher order provider for running multiple tasks in a sequence.
 .TP
-\fBedoc\fR 
+\fBedoc\fR
 Generate documentation using edoc.
 .TP
-\fBescriptize\fR 
+\fBescriptize\fR
 Generate escript archive.
 .TP
-\fBeunit\fR [\fI--app\fR] [\fI--application\fR] [\fI-c\fR|\fI--cover\fR] [\fI--cover_export_name\fR] [\fI-d\fR|\fI--dir\fR] [\fI-f\fR|\fI--file\fR] [\fI-m\fR|\fI--module\fR] [\fI-s\fR|\fI--suite\fR] [\fI-v\fR|\fI--verbose\fR] [\fI--name\fR] [\fI--sname\fR] [\fI--setcookie\fR] 
+\fBeunit\fR [\fI--app\fR] [\fI--application\fR] [\fI-c\fR|\fI--cover\fR] [\fI--cover_export_name\fR] [\fI-d\fR|\fI--dir\fR] [\fI-f\fR|\fI--file\fR] [\fI-m\fR|\fI--module\fR] [\fI-s\fR|\fI--suite\fR] [\fI-v\fR|\fI--verbose\fR] [\fI--name\fR] [\fI--sname\fR] [\fI--setcookie\fR]
 Run EUnit Tests.
 .IP
 \fI--app\fR: Comma separated list of application test suites to run. Equivalent to `[{application, App}]`.
@@ -195,18 +193,18 @@ Run EUnit Tests.
 .IP
 \fI--setcookie\fR: Sets the cookie if the node is distributed
 .TP
-\fBget-deps\fR 
+\fBget-deps\fR
 Fetch dependencies.
 .TP
-\fBhelp\fR 
+\fBhelp\fR
 Display a list of tasks or help for a given task or subtask.
 .TP
-\fBnew\fR [\fI-f\fR|\fI--force\fR] 
+\fBnew\fR [\fI-f\fR|\fI--force\fR]
 Create new project from templates.
 .IP
 \fI--force\fR: overwrite existing files
 .TP
-\fBpath\fR [\fI--app\fR] [\fI--base\fR] [\fI--bin\fR] [\fI--ebin\fR] [\fI--lib\fR] [\fI--priv\fR] [\fI-s\fR|\fI--separator\fR] [\fI--src\fR] [\fI--rel\fR] 
+\fBpath\fR [\fI--app\fR] [\fI--base\fR] [\fI--bin\fR] [\fI--ebin\fR] [\fI--lib\fR] [\fI--priv\fR] [\fI-s\fR|\fI--separator\fR] [\fI--src\fR] [\fI--rel\fR]
 Print paths to build dirs in current profile.
 .IP
 \fI--app\fR: Comma separated list of applications to return paths for.
@@ -227,10 +225,10 @@ Print paths to build dirs in current pro
 .IP
 \fI--rel\fR: Return the `rel' path of the current profile.
 .TP
-\fBpkgs\fR 
+\fBpkgs\fR
 List information for a package.
 .TP
-\fBrelease\fR [\fI-n\fR|\fI--relname\fR] [\fI-v\fR|\fI--relvsn\fR] [\fI-g\fR|\fI--goal\fR] [\fI-u\fR|\fI--upfrom\fR] [\fI-o\fR|\fI--output-dir\fR] [\fI-h\fR|\fI--help\fR] [\fI-l\fR|\fI--lib-dir\fR] [\fI-p\fR|\fI--path\fR] [\fI--default-libs\fR] [\fI-V\fR|\fI--verbose\fR] [\fI-d\fR|\fI--dev-mode\fR] [\fI-i\fR|\fI--include-erts\fR] [\fI-a\fR|\fI--override\fR] [\fI-c\fR|\fI--config\fR] [\fI--overlay_vars\fR] [\fI--vm_args\fR] [\fI--sys_config\fR] [\fI--system_libs\fR] [\fI--version\fR] [\fI-r\fR|\fI--root\fR] 
+\fBrelease\fR [\fI-n\fR|\fI--relname\fR] [\fI-v\fR|\fI--relvsn\fR] [\fI-g\fR|\fI--goal\fR] [\fI-u\fR|\fI--upfrom\fR] [\fI-o\fR|\fI--output-dir\fR] [\fI-h\fR|\fI--help\fR] [\fI-l\fR|\fI--lib-dir\fR] [\fI-p\fR|\fI--path\fR] [\fI--default-libs\fR] [\fI-V\fR|\fI--verbose\fR] [\fI-d\fR|\fI--dev-mode\fR] [\fI-i\fR|\fI--include-erts\fR] [\fI-a\fR|\fI--override\fR] [\fI-c\fR|\fI--config\fR] [\fI--overlay_vars\fR] [\fI--vm_args\fR] [\fI--sys_config\fR] [\fI--system_libs\fR] [\fI--version\fR] [\fI-r\fR|\fI--root\fR]
 Build release of project.
 .IP
 \fI--relname\fR: Specify the name for the release that will be generated
@@ -273,7 +271,7 @@ Build release of project.
 .IP
 \fI--root\fR: The project root directory
 .TP
-\fBrelup\fR [\fI-n\fR|\fI--relname\fR] [\fI-v\fR|\fI--relvsn\fR] [\fI-g\fR|\fI--goal\fR] [\fI-u\fR|\fI--upfrom\fR] [\fI-o\fR|\fI--output-dir\fR] [\fI-h\fR|\fI--help\fR] [\fI-l\fR|\fI--lib-dir\fR] [\fI-p\fR|\fI--path\fR] [\fI--default-libs\fR] [\fI-V\fR|\fI--verbose\fR] [\fI-d\fR|\fI--dev-mode\fR] [\fI-i\fR|\fI--include-erts\fR] [\fI-a\fR|\fI--override\fR] [\fI-c\fR|\fI--config\fR] [\fI--overlay_vars\fR] [\fI--vm_args\fR] [\fI--sys_config\fR] [\fI--system_libs\fR] [\fI--version\fR] [\fI-r\fR|\fI--root\fR] 
+\fBrelup\fR [\fI-n\fR|\fI--relname\fR] [\fI-v\fR|\fI--relvsn\fR] [\fI-g\fR|\fI--goal\fR] [\fI-u\fR|\fI--upfrom\fR] [\fI-o\fR|\fI--output-dir\fR] [\fI-h\fR|\fI--help\fR] [\fI-l\fR|\fI--lib-dir\fR] [\fI-p\fR|\fI--path\fR] [\fI--default-libs\fR] [\fI-V\fR|\fI--verbose\fR] [\fI-d\fR|\fI--dev-mode\fR] [\fI-i\fR|\fI--include-erts\fR] [\fI-a\fR|\fI--override\fR] [\fI-c\fR|\fI--config\fR] [\fI--overlay_vars\fR] [\fI--vm_args\fR] [\fI--sys_config\fR] [\fI--system_libs\fR] [\fI--version\fR] [\fI-r\fR|\fI--root\fR]
 Create relup of releases.
 .IP
 \fI--relname\fR: Specify the name for the release that will be generated
@@ -316,10 +314,10 @@ Create relup of releases.
 .IP
 \fI--root\fR: The project root directory
 .TP
-\fBreport\fR 
+\fBreport\fR
 Provide a crash report to be sent to the rebar3 issues page.
 .TP
-\fBshell\fR [\fI--config\fR] [\fI--name\fR] [\fI--sname\fR] [\fI--setcookie\fR] [\fI--script\fR] [\fI--apps\fR] [\fI--start-clean\fR] [\fI--user_drv_args\fR] 
+\fBshell\fR [\fI--config\fR] [\fI--name\fR] [\fI--sname\fR] [\fI--setcookie\fR] [\fI--script\fR] [\fI--apps\fR] [\fI--start-clean\fR] [\fI--user_drv_args\fR]
 Run shell with project apps and deps in path.
 .IP
 \fI--config\fR: Path to the config file to use. Defaults to {shell, [{config, File}]} and then the relx sys.config file if not specified.
@@ -338,7 +336,7 @@ Run shell with project apps and deps in
 .IP
 \fI--user_drv_args\fR: Arguments passed to user_drv start function for creating custom shells.
 .TP
-\fBtar\fR [\fI-n\fR|\fI--relname\fR] [\fI-v\fR|\fI--relvsn\fR] [\fI-g\fR|\fI--goal\fR] [\fI-u\fR|\fI--upfrom\fR] [\fI-o\fR|\fI--output-dir\fR] [\fI-h\fR|\fI--help\fR] [\fI-l\fR|\fI--lib-dir\fR] [\fI-p\fR|\fI--path\fR] [\fI--default-libs\fR] [\fI-V\fR|\fI--verbose\fR] [\fI-d\fR|\fI--dev-mode\fR] [\fI-i\fR|\fI--include-erts\fR] [\fI-a\fR|\fI--override\fR] [\fI-c\fR|\fI--config\fR] [\fI--overlay_vars\fR] [\fI--vm_args\fR] [\fI--sys_config\fR] [\fI--system_libs\fR] [\fI--version\fR] [\fI-r\fR|\fI--root\fR] 
+\fBtar\fR [\fI-n\fR|\fI--relname\fR] [\fI-v\fR|\fI--relvsn\fR] [\fI-g\fR|\fI--goal\fR] [\fI-u\fR|\fI--upfrom\fR] [\fI-o\fR|\fI--output-dir\fR] [\fI-h\fR|\fI--help\fR] [\fI-l\fR|\fI--lib-dir\fR] [\fI-p\fR|\fI--path\fR] [\fI--default-libs\fR] [\fI-V\fR|\fI--verbose\fR] [\fI-d\fR|\fI--dev-mode\fR] [\fI-i\fR|\fI--include-erts\fR] [\fI-a\fR|\fI--override\fR] [\fI-c\fR|\fI--config\fR] [\fI--overlay_vars\fR] [\fI--vm_args\fR] [\fI--sys_config\fR] [\fI--system_libs\fR] [\fI--version\fR] [\fI-r\fR|\fI--root\fR]
 Tar archive of release built of project.
 .IP
 \fI--relname\fR: Specify the name for the release that will be generated
@@ -381,24 +379,24 @@ Tar archive of release built of project.
 .IP
 \fI--root\fR: The project root directory
 .TP
-\fBtree\fR [\fI-v\fR|\fI--verbose\fR] 
+\fBtree\fR [\fI-v\fR|\fI--verbose\fR]
 Print dependency tree.
 .IP
 \fI--verbose\fR: Print repo and branch/tag/ref for git and hg deps
 .TP
-\fBunlock\fR 
+\fBunlock\fR
 Unlock dependencies.
 .TP
-\fBupdate\fR 
+\fBupdate\fR
 Update package index.
 .TP
-\fBupgrade\fR 
+\fBupgrade\fR
 Upgrade dependencies.
 .TP
-\fBversion\fR 
+\fBversion\fR
 Print version for rebar and current Erlang.
 .TP
-\fBxref\fR 
+\fBxref\fR
 Run cross reference analysis.
 
 .SH ENVIRONMENT
@@ -435,7 +433,7 @@ Name of rebar configuration files. Defau
 
 .TP
 \fBREBAR_GIT_CLONE_OPTIONS\fR
-Arguments to add after each \fIgit clone\fR operation. For example, the value \fI--reference ~/.cache/repos.reference\fR allows to create a cache of all fetched repositories across builds
+Arguments to add after each \fIgit clone\fR operation. For example, the value \fI--reference ~/.cache/repos.reference\fR allows one to create a cache of all fetched repositories across builds
 
 .SH Configuration File Options
 See \fIhttp://www.rebar3.org/v3.0/docs/configuration\fR
